2020/dec12/dec12_test.go

```diff
@@ -9,5 +9,11 @@ import (
 func TestDec12_Solve(t *testing.T) {
 	s, err := New("testdata/in.txt").Solve()
 	assert.NoError(t, err)
-	assert.Contains(t, s, "")
+	// assert.Contains(t, s, "Manhattan distance is 25.")
+	assert.Contains(t, s, "Manhattan distance is 286.")
```

Comment:
these two lines are strange - maybe you can set a variable to let the reader know why one is commented